ksp_plugin/plugin.cpp

```diff
@@ -363,6 +371,9 @@ void Plugin::AdvanceTime(Instant const& t, Angle const& planetarium_rotation) {
   CHECK(!initializing_);
   CHECK_GT(t, current_time_);
   FreeVessels();
+  for (auto const& id_vessel : vessels_) {
```

Comment:
Preference for calling the loop variable `pair` and for declaring something like:
```
Vessel const& vessel = *pair.second;
```